packages/components/src/FilterDatePicker/types.ts

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { FieldMessageStatus } from "@kaizen/draft-form"
+import { StringSuggestions } from "~types/StringSuggestions"
+
+export type { DateRange, DisabledDays } from "@kaizen/date-picker/src/types"
```

Comment:
Quite a few types in this file are the same as the ones in FilterDRP. Since these all get exported to the KAIO entrypoint, I think it will actually create conflicts 😅

Reply:
Ah yeah bingo. When I tried to export FilterDRP is complained haha, I'll consolidate.

Reply:
Hmm do we need to actually export these...?

Reply:
It's always better to export the types imo. When I was in product land, I often wanted to use the types from a DS component because I was either defining something to pass into the component, or maybe using it within a util function.

Reply:
Yeah but these types are part of the `date-picker` so they can get it from there like we are?

Reply:
You want people to import the `date-picker` package even when they just want to use `FilterD(R)P` and only have KAIO imported? 😞

Reply:
Hmm... I suppose not, despite that scenario being pretty niche right now lol